CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 # unreleased
 
 - Add typescript types
+- Dependency Updates:
```

Comment:
this is a breaking change, is it not?

Reply:
After some discussion and research, we decided that this should in fact be a major version update. Since we updated node on all of our modules, we're going to do some beta deploys to verify that updating them to v18 of node does not break anything in braintree-web.